### PR TITLE
added stars

### PR DESCRIPTION
**Pull request by Jason-Mollerup**
added bar click -> adds star of bar to filter criteria
added remove star filter on bar click and filter click

### CODE REVIEW

File: client/dist/styles/ratingsReviews.css [hunk 1]
```diff
@@ -308,4 +345,36 @@
...
+
+.stars {
+  position: relative;
+  vertical-align: middle;
+  display: inline-block;
+  color: lightgray;
+  overflow: hidden;
+}
+
+.full-stars {
```
Comment (callumreid) [inline client/dist/styles/ratingsReviews.css:360]: Curious as to why stars class is relative position while full stars is absolute? Not to come at the CSS King, just wondering how relative and absolute positioning work together if screen size changes

File: client/src/RatingsAndReviews/RatingsBreakdown/BreakdownClickable.jsx [hunk 2]
```diff
@@ -17,7 +17,8 @@ const BreakdownClickable = (props) => {
 BreakdownClickable.propTypes = {
   stars: PropTypes.number,
   percent: PropTypes.number,
-  count: PropTypes.number
+  count: PropTypes.number,
+  starFilterClick: PropTypes.func
```
Comment (callumreid) [inline client/src/RatingsAndReviews/RatingsBreakdown/BreakdownClickable.jsx:21]: I notice that you have propTypes camelCased on line 17 but capitalized both words just below. I've been using syntax like 
`BreakdownClickable.propTypes = {
  stars: propTypes.number,
  percent: propTypes.number
}
`.
Not sure if it prop-types works the same either way, but if it does, I'll adjust mine to match your syntax

Comment (Jason-Mollerup) [reply client/src/RatingsAndReviews/RatingsBreakdown/BreakdownClickable.jsx:21]: yeah that is just the way I saw it in the docs. 

.propTypes for adding propTypes to a class / functional component 
vs 
PropTypes for accessing the react PropTypes class

**Review by quinn-lima — APPROVED** [review]

**Review by callumreid — COMMENTED** [review]
Excited to get to throw real star data into the mix. I know at least I will have to make sure I have access not only to the Stars function, but also to the data for star ratings in my component. For each relatedProduct, I'll need to get the average star rating. This might already be accessible as I haven't into it yet, but something like averageStarRatingForCurrentItem would make sense as app-level state property

File: client/src/RatingsAndReviews/RatingsBreakdown/BreakdownClickable.jsx (hunk 2 above)
Comment (callumreid) [reply client/src/RatingsAndReviews/RatingsBreakdown/BreakdownClickable.jsx:21]: ahh gotcha, thanks for clarification!

**Pull request merged**